weave-js/src/components/Panel2/PanelString.tsx

```diff
@@ -224,16 +245,19 @@ export const PanelString: React.FC<PanelStringProps> = props => {
         console.error(e);
       }
     }
+    const textStyle: React.CSSProperties = isRTL(fullStr)
```

Comment:
we could also limit the regex check to just a portion of the str if its gigantic. Like im just imaging tables of novels or something, you can probably tell if its hebrew and should be RTL from just the first... 1000 chars? Idk

Reply:
Yeah I'm limiting to first 100